test/blackbox-tests/test-cases/preprocessor-directory.t/run.t

```diff
@@ -20,4 +20,4 @@ file which specifies the preprocessor is located there:
   (lang dune 3.8)
   (package (name foo))
   $ dune exec ./src/foo.exe
-  bar (from the rewrite.sh script in the src directory)
```

Comment:
can you add a note that this behaviour is not intended, and add a link to https://github.com/ocaml/dune/issues/7043